src/CreateDisc.js

```diff
@@ -0,0 +1,121 @@
+import React, { Component } from 'react';
+import { getManufacturers, createDisc } from './fetch-utils';
+import './CreateDisc.css';
+import classNames from 'classnames';
+class CreateDisc extends Component {
+    state = { 
+        disc: '',
+        speed: 0,
+        type: '',
+        stable: true,
+        manufacturer: 'Innova',
+        manufacturers: [],
+        message: '',
+        error: false,
+    };
+    componentDidMount = async () => {
+        const manufacturers = await getManufacturers();
+        this.setState({ manufacturers});
+    };
+    getManufacturerId = () => {
+        const manufacturersObj = this.state.manufacturers.find(
+            (mn) => mn.name === this.state.manufacturer
+        );
+        return manufacturersObj.id;
+    };
+
+    handleCreateDisc = async (e) => {
+        e.preventDefault();
+        const discData = {
+            disc: this.state.disc,
+            speed: this.state.speed,
+            type: this.state.type,
+            manufacturer_id: this.getManufacturerId(),
+            stable: this.state.stable,
+        }
+        const data = await createDisc(discData);
+        if (data.error) {
+            this.setState({ message: data.error, error: true});
+        } else {
+            this.props.history.push('/');
+        }
+    };
+    render() { 
+        return (
+            <>
+                {this.state.message && (
+                    <div 
+                        className={classNames({
+                            message: true,
+                            error: this.state.error,
+                            success: !this.state.error,
+                        })}
+                    >{this.state.message}</div>
+                )}
+                <h1>{this.state.disc}</h1>
+                <form id='create-disc'>
+                    <div className='form-group'>
```

Comment:
I love how you are grouping your form elements together! Instead of using a `<div className='form-group'>` you can actually use another form element called `<fieldset>`! It's super handy for grouping together form elements and you can even add custom titles to each group with the `<legend>` tag within the `<fieldset>`. 

Reference: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset